pybikes/fsm.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017, aronsky <aronsky@gmail.com>
+
+from lxml import etree
+
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes import utils
+
+
+class FSMSystem(BikeShareSystem):
+    sync = True
+    unifeed = True
+
+    meta = {
+        'system': 'fsm',
+        'company': ['FSM Ground Services Ltd.']
+    }
+
+    def __init__(self, tag, meta, feed_url):
+        super(FSMSystem, self).__init__(tag, meta)
+
+        self.feed_url = feed_url
+
+    def update(self, scraper=None):
+        scraper = scraper or utils.PyBikesScraper()
+
+        stations = []
+
+        data = scraper.request(self.feed_url)
+        dom = etree.fromstring(data.encode('utf-8'))
+        stations = self.get_stations(dom)
+        self.stations = list(stations)
+
+    def get_stations(self, dom):
+        ns = {'kml': 'http://www.opengis.net/kml/2.2'}
+        for placemark in dom.xpath('//kml:Placemark', namespaces=ns):
+            name = placemark.findtext('kml:name', namespaces=ns)
+            info = placemark.findtext('kml:description', namespaces=ns)
+            station_uid, bikes, free = utils.re.findall(r'\w+\:\s*(\d+)', info)
```

Comment:
Got this:

```
Testing fsm
=============================
Testing u'Tel-O-Fun', u"Tel Aviv-Yaffo, Giv'atayim, Ramat-Gan"
Tel-O-Fun has 206 stations
[▚] testing 1F
======================================================================
FAIL: test_fsm (__main__.TestSystems)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/unittest_pybikes.py", line 367, in test_schema
    self._test_systems(schema)
  File "tests/unittest_pybikes.py", line 27, in _test_systems
    self._test_system(instance['tag'], key)
  File "tests/unittest_pybikes.py", line 47, in _test_system
    self._test_station_ok(p_sys, p_sys.stations[i])
  File "tests/unittest_pybikes.py", line 70, in _test_station_ok
    self.assertIsInstance(station.bikes, int)
AssertionError: '4' is not an instance of <type 'int'>
```

Thought we were casting to int on constructor, but seems not. bikes and free must be `int`s.